modules/ddl-and-loading/pages/modifying-a-graph-schema.adoc

```diff
@@ -29,15 +29,25 @@ Load or query operations which begin before the schema change will be completed
 
 == *Global vs. Local Schema Changes*
 
-Only a superuser or globaldesigner can add, alter, or drop global vertex types or global edge types, which are those that are created using CREATE VERTEX or CREATE ... EDGE.  This rule applies even if the vertex or edge type is used in only one graph. To make these changes, the user uses a GLOBAL SCHEMA_CHANGE JOB.
+Only a superuser or globaldesigner can add, alter, or drop global vertex types or global edge types, which are those that are created using `CREATE VERTEX` or `CREATE` ... `EDGE`.  This rule applies even if the vertex or edge type is used in only one graph. To make these changes, the user uses a `GLOBAL` `SCHEMA_CHANGE JOB`.
```

Comment:
Whenever we see in the documentaion "only a superuser ..", that actually is no longer the case since 3.2.

Since 3.2, our access control is privilege-based. Users can create their own roles and add the corresponding privileges. They can create a role with all superuser privileges and call it by a different name. Every time we see things described this way, we should change the wording to reflect that. 

Only users with the `WRITE_SCHEMA` privilege on the global scope can add, alter ...